specification/synapse/data-plane/Microsoft.Synapse/preview/2019-11-01-preview/sparkFrontend.json

```diff
@@ -1007,6 +1007,13 @@
             "type": "string"
           }
         },
+        "pyFiles": {
```

Comment:
>"pyFiles": { [](start = 8, length = 12)

Could you please help me understand why we need add this property to LivyRequestBase? The others make sense to me

Reply:
@aim-for-better, please check the internal Spark Frontend Swagger specs